keps/0014-pull-request-process.md

```diff
@@ -0,0 +1,129 @@
+* Not be limited by public CI services (queue times, concurrency, resources).
+* Use CNCF-standard tooling for our processes.
+
+## Proposal
+
+A Prow cluster will be deployed in Kubernetes for running CI and PR/issue automation. This cluster lives at https://prow.toolsinfra.mesosphe.re/ and will take action on Github as the `ci-mergebot` user. KEP-0004 outlines the infrastructure required.
+
+### Issues and pull requests
+
+Prow will be used to manage issue and pull request labels and life cycle. When an issue or pull request is created or commented on, it will be scanned for Prow commands (lines that begin with `/`).
+
+Supported commands:
+
+* `/help`: labels the issue or PR as wanting help from external contributors.
+* `/good-first-issue`: labels the issue or PR as a good issue for new contributors to tackle.
+* `/priority`: sets the priority on the issue or PR.
+* `/milestone`: set a milestone for the issue or PR.
+* `/lifecycle`: set a specific lifecycle label on the issue or PR.
+* `/kind`: label the issue or PR as a specific kind (e.g., bug, enhancement, etc).
+* `/close`: close an issue or PR .
+* `/repoen`: open an issue on PR that has been closed.
+
+When issues and PRs are 30 days old, they will be labeled `lifecycle/rotten` and closed automatically at 60.
+
+### Pull requests
+
+For pull requests only, Prow will automatically request review from the proper users, control when tests are run (and run them), report test status, and automatically merge the pull request when tests are passed and it has the proper approvals.
+
+Supported commands:
+
+* `/ok-to-test`: if the PR creator is an external contributor, Prow will not run tests on the pull request until a contributor has specified `/ok-to-test`.
+* `/test`: re-run all of the tests or a specific test.
+* `/retest`: re-run failed tests.
+* `/override`: ignore a test's status.
+* `/approve`: approve a pull request.
+* `/lgtm`: mark a pull request as ready for merge, once it has the required number of approvals and all tests are passed, Prow will merge it automatically.
+* `/cc`: request review from a specific user.
+* `/hold`: prevent a pull request from being merged.
+
+Once two contributors have marked `/approve` (or `/lgtm`, which implies `/approve`) and one types `/lgtm` and all tests pass, a pull request will be automatically merged.
+
+Contributors are defined in `OWNERS` files in the repository. The `OWNERS` file is a YAML file containing valid approvers and reviewers:
+
+```
+approvers:
+- user1
+
+reviewers:
+- user2
+```
+
+Users listed in `approvers` are allowed to approve a pull request. Users listed in `reviewers` are eligible for being automatically requested as reviewers by Prow on new pull requests (Prow will select two users from the list at random).
+
+Prow will look for `OWNERS` files covering the directories being changed, e.g., the root `OWNERS` file applies to all changes, but `./tests/OWNERS` only applies to changes that in `./tests/`.
```

Comment:
Whats the difference between `OWNERS` and/or the one we have already as `CODEOWNERS` ? https://github.com/kudobuilder/kudo/blob/master/.github/CODEOWNERS

Reply:
OWNERS is understood by Prow, whereas CODEOWNERS is not